RecoHGCal/TICL/python/TICLSeedingRegions_cff.py

```diff
@@ -5,13 +5,13 @@
 # SEEDING REGION
 
 ticlSeedingGlobal = _ticlSeedingRegionProducer.clone(
-  algoId = 2
+  seedingPSet = _ticlSeedingRegionProducer.seedingPSet.clone(type=cms.string("SeedingRegionGlobal"))
```

Comment:
Would this work here?
```suggestion
  seedingPSet = _ticlSeedingRegionProducer.seedingPSet.clone(type="SeedingRegionGlobal")
```
(we are cleaning up all configs to avoid specifying the types in multiple places when it can be avoided)

Reply:
yes it worked
do you want this in 11_1_X also? or can we leave it there as it is? we have a backport PR of this here: https://github.com/cms-sw/cmssw/pull/33028

Reply:
I think normally backports and the original PRs should be as close as possible. So yes, I think you need to implement it there, too.

Reply:
okay, now it's done in 11_1_X PR as well